Makefile

```diff
@@ -97,14 +97,16 @@ build-lambdas:
 	@echo -e "\n\n\n============================="
 	@echo "Install Node Modules in all packages"
 	@echo "============================="
-	npm install --production --prefix server
+	npm install --prefix server
 	npm install --production --prefix server/lambda/layer/common/nodejs/custom_modules
```

Comment:
custom_modules has no package definition. I doubt this is effective?

Reply:
It does have package.json file if that is what you mean.